web/public/javascripts/dashboard.js

```diff
@@ -326,6 +326,7 @@
               }
             }
           });
+
```

Comment:
Please remove this new line, you didn't change this file. This may cause further merge conflict.